.pre-commit-config.yaml

```
@@ -0,0 +1,13 @@
+- repo: https://github.com/pre-commit/mirrors-eslint
```

```suggestion
- repo: https://github.com/pre-commit/pre-commit-hooks
  rev: v2.3.0
  hooks:
    - id: check-added-large-files
    - id: check-case-conflict
    - id: check-executables-have-shebangs
    - id: check-json
    - id: check-vcs-permalinks
    - id: check-yaml
    - id: end-of-file-fixer
    - id: trailing-whitespace
- repo: https://github.com/pre-commit/mirrors-eslint
```
Addition of these common hooks will resolve many observed git frictions. In particular, the combination of `end-of-file-fixer` and `trailing-whitespace` would resolve the extra chars in the `.pre-commit-config.yaml` itself.